db/seeds/07_animations.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animations (type)
+VALUES ('rain'),
```

Comment:
can we include 'none' as the first/default value for animations?

Reply:
if it's "none" then we can just make animation_id null in the theme_id table